Package.swift

```diff
@@ -1,18 +1,21 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.1
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "Mailgun",
+    platforms: [
+       .macOS(.v10_15)
```

Comment:
Let's drop this to the lowest possible macOS version to avoid making breaking changes later.

Reply:
Done ✅